RELEASING.md

```diff
@@ -0,0 +1,40 @@
+# Release Process
+
+## Preparing for a release
+
+1. Ensure master docs are passing semaphore tests.
+
+2. Check out latest master locally.
+
+## Creating the release
+
+1. Run `python release-scripts/do_release.py`, which will walk through creation of the necessary
+release directories, and will replace references to nightly artifacts with release ones.
+
+2. Fix calico-cni binary links which are not updated by the release script (see [#147](https://github.com/projectcalico/calico/issues/147)).
```

Comment:
Is there a command we can include that will find all of these links for someone going through this guide?